### PR TITLE
fix: always replacing placeholders

### LINKED ISSUE #31625: import placeholders should be converted to an option
### Description

The [docs](https://www.keycloak.org/server/importExport) mention using placeholders, but that only applies to import on startup.

### Discussion

_No response_

### Motivation

The operator, and possibly other users, may want to run import with placeholders with a regular import command and should be able to do so without using a system property.

### Details

_No response_

### PR DESCRIPTION
**Pull request by shawkins**
closes: #31625

### CODE REVIEW

File: docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc [hunk 1]
```diff
@@ -97,3 +97,9 @@ This is enforced by default, and can be disabled using the SPI option `spi-singl
 It used to be difficult to regain access to a {project_name} instance when all admin users were locked out. The process required multiple advanced steps, including direct database access and manual changes. In an effort to improve the user experience, {project_name} now provides multiple ways to bootstrap a new admin account, which can be used to recover from such situations.
 
 Consequently, the environment variables `KEYCLOAK_ADMIN` and `KEYCLOAK_ADMIN_PASSWORD` have been deprecated. You should use `KC_BOOTSTRAP_ADMIN_USERNAME` and `KC_BOOTSTRAP_ADMIN_PASSWORD` instead. These are also general options, so they may be specified via the cli or other config sources, for example `--bootstrap-admin-username=admin`. For more information, see the new https://www.keycloak.org/server/bootstrap-admin-recovery[Bootstrap admin and recovery] guide.
+
+= kc import placeholder replacement
```
Comment (mabartos) [inline docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc:101]:
```suggestion
= CLI import placeholder replacement
```
I'd prefer this.

File: docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc [hunk 2]
```diff
@@ -97,3 +97,9 @@ This is enforced by default, and can be disabled using the SPI option `spi-singl
...
+
+kc.[sh|bat] import now has placeholder replacement enabled. Previously placeholder replacement was only enabled for realm import at startup.
```
Comment (mabartos) [inline docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc:103]:
```suggestion
CLI command `kc.[sh|bat] import` now has placeholder replacement enabled. Previously placeholder replacement was only enabled for realm import at startup. 
```

**Review by mabartos — COMMENTED** [review]
@shawkins Just a few nitpick suggestions. Otherwise, LGTM.

**Comment by mabartos** [comment]
Some conflict there; otherwise, LGTM.

**Review by mabartos — DISMISSED** [review]

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@shawkins Thank you for the changes! We should probably also update [this](https://github.com/keycloak/keycloak/blob/e7d9a717c9cf4d3b0089af3618f78efce15b38db/docs/guides/server/importExport.adoc?plain=1#L114) chapter in the guide to be generic for the `import` command as well as startup.

**Comment by shawkins** [comment]
> @shawkins Thank you for the changes! We should probably also update [this](https://github.com/keycloak/keycloak/blob/e7d9a717c9cf4d3b0089af3618f78efce15b38db/docs/guides/server/importExport.adoc?plain=1#L114) chapter in the guide to be generic for the `import` command as well as startup.

Quite right, I had missed that it was specific to startup. Corrected now.

File: docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc (hunk 1 above)
Comment (mabartos) [reply docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc:137]: It seems the changes were reverted.

Comment (shawkins) [reply docs/documentation/upgrading/topics/changes/changes-26_0_0.adoc]: Applied again.

**Review by mabartos — DISMISSED** [review]

**Review by vmuzikar — DISMISSED** [review]

**Comment by vmuzikar** [comment]
Realized we can also remove:
https://github.com/keycloak/keycloak/blob/242796b89679933ae8df63953960308eef5afb07/operator/src/main/java/org/keycloak/operator/controllers/KeycloakRealmImportJobDependentResource.java#L152

But we can do it as a follow-up.

**Comment by shawkins** [comment]
> Realized we can also remove:
> 
> https://github.com/keycloak/keycloak/blob/242796b89679933ae8df63953960308eef5afb07/operator/src/main/java/org/keycloak/operator/controllers/KeycloakRealmImportJobDependentResource.java#L152
> 
> But we can do it as a follow-up.

Updated fine to remove that with this pr - since the feature just went into 26 there's no concern about a behavioral change.

**Review by vmuzikar — APPROVED** [review]

**Pull request merged**